src/Dialect/Mlir/DialectBuilder.cpp

```diff
@@ -653,6 +654,20 @@ void SCFBuilder::ifThenElse(Value cond,
   }
 }
 
+void SCFBuilder::parallelLoop(ValueRange lowerBounds, ValueRange upperBounds,
```

Comment:
Out of curiosity, how does it do multiple parallel loops? Does it do an openmp collapse, or simply parallel of parallel ..... of parallel?

Reply:
currently it just lowers to regular branching code